### PR TITLE
improved README.md

### PR DESCRIPTION
**Pull request by kartik-raj7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,9 +39,9 @@ src="https://github.com/common-workflow-language/cwl-website/raw/main/openstand-
 The Common Workflow Language (CWL) is an open standard for describing analysis
 workflows and tools in a way that makes them portable and scalable across a
 variety of software and hardware environments, from workstations to cluster,
-cloud, and high performance computing (HPC) environments.  CWL is designed to
-meet the needs of data-intensive science, such as Bioinformatics, Medical
-Imaging, Astronomy, High Energy Physics, and Machine Learning.
+cloud, and high performance computing (HPC) environments.CWL is designed to
```
Comment (tetron) [inline README.md:42]: You've introduced a typo, please fix.

Comment (kartik-raj7) [reply README.md:42]: Ok

Comment (tetron) [reply README.md:42]: I mean you removed the space between "environments.CWL"

Comment (kartik-raj7) [reply README.md]: Done

**Comment by tetron** [comment]
I'm afraid this PR doesn't meet our quality standards.

**Pull request closed without merging**